app/models/package_manager/rubygems.rb

```diff
@@ -108,5 +107,53 @@ def self.download_registry_users(name)
     def self.registry_user_url(login)
       "https://rubygems.org/profiles/#{login}"
     end
+
+    def self.parse_html_versions(raw_project)
```

Comment:
Am I correct that this method is only used to collect yanked versions, and we'll never retrieve anything not yanked from this? We'll get everything that isn't yanked from `parse_json_versions`? If so, it might be nice to have some comments and/or renamings to make that clear.

Reply:
Correct, since the only reason to use the HTML option would be to collect yanked versions. I updated the method name to clarify this.